qubesadmin/vm/__init__.py

```diff
@@ -428,6 +428,21 @@ def cleanup(self):
             except qubesadmin.exc.QubesVMNotRunningError:
                 pass
 
+    def start(self):
+        """Start this DispVM"""
+        if self.app.qubesd_connection_type == 'socket':
```

Comment:
I don't think this condition is necessary in `start()` method. It's used in `run_service()` because when running outside of dom0 the DispVM will be created automatically on a service call. But it isn't the case for just starting a DispVM.